scripts/sort-makefile-am-sources.pl

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/perl
+
+while (<>) {
+    # print "read1: $_";
+    if (! /^(nodist_)?tests_test.*_SOURCES/) {
```

Comment:
The sorting style guideline should apply to all SOURCES lists in all Makefile.am

Please extend this to sort the non-test lists.

Reply:
done that. In the process I also discovered we were including a file twice in squid_SOURCES